### PR TITLE
Fix issues on history page

### PR DESCRIPTION
**Pull request by janithRS**

### CODE REVIEW

**Comment by mohamednizar** [comment]
@janithRS  please Remove the log files in the pull request.

File: storage/logs/laravel-2020-02-19.log [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[2020-02-19 06:54:08] local.ERROR: Command "mirgrate" is not defined.
```
Comment (mohamednizar) [inline storage/logs/laravel-2020-02-19.log]: Remove this file

File: storage/logs/laravel-2020-03-03.log [hunk 2]
```diff
@@ -0,0 +1,57 @@
+[2020-03-03 02:53:04] local.ERROR: There are no commands defined in the "update" namespace. {"exception":"[object] (Symfony\\Component\\Console\\Exception\\NamespaceNotFoundException(code: 0): There are no commands defined in the \"update\" namespace. at /app/vendor/symfony/console/Application.php:592)
```
Comment (mohamednizar) [inline storage/logs/laravel-2020-03-03.log]: remove this file

File: storage/logs/laravel-2020-03-05.log [hunk 3]
```diff
@@ -0,0 +1,93 @@
+[2020-03-05 09:13:57] local.ERROR: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'openemis.config items' doesn't exist (SQL: insert into `config items` (`name`, `code`) values (security_timeout, 2020-03-05 09:13:57 am)) {"exception":"[object] (Illuminate\\Database\\QueryException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'openemis.config items' doesn't exist (SQL: insert into `config items` (`name`, `code`) values (security_timeout, 2020-03-05 09:13:57 am)) at /app/vendor/laravel/framework/src/Illuminate/Database/Connection.php:664, Doctrine\\DBAL\\Driver\\PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'openemis.config items' doesn't exist at /app/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:63, PDOException(code: 42S02): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'openemis.config items' doesn't exist at /app/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:61)
```
Comment (mohamednizar) [inline storage/logs/laravel-2020-03-05.log]: remove this file

**Comment by janithRS** [comment]
Removed the logs

**Review by mohamednizar — APPROVED** [review]

**Pull request merged**